pkg/minishift/openshift/service.go

```diff
@@ -86,12 +103,20 @@ func GetServiceURLs(serviceListNamespace string) ([]ServiceURL, error) {
 
 	// iterate over namespaces, get command output, format route in ServiceURL
 	for _, namespace := range namespaces {
-		outputData, err := getServiceURLsOutput(namespace)
+		serviceNodePort, err := getService(namespace, ip)
```

Comment:
So now you make requests per namespace. You see not issue with this anymore. We talked about getting all service info in one go. Just wondering and  trying to understand your reasoning.